entity-framework/core/managing-schemas/migrations/index.md

```diff
@@ -1,294 +1,188 @@
 ---
-title: Migrations - EF Core
+title: Migrations Overview - EF Core
 author: bricelam
 ms.author: bricelam
-ms.date: 10/05/2018
+ms.date: 05/06/2020
 uid: core/managing-schemas/migrations/index
 ---
-# Migrations
+# Migrations Overview
 
-A data model changes during development and gets out of sync with the database. You can drop the database and let EF create a new one that matches the model, but this procedure results in the loss of data. The migrations feature in EF Core provides a way to incrementally update the database schema to keep it in sync with the application's data model while preserving existing data in the database.
+In real life projects, data models change as features get implemented: new entities or properties are added and removed, and databases schema needs to be changed accordingly to be kept in sync with the application. The migrations feature in EF Core provides a way to incrementally update the database schema to keep it in sync with the application's data model while preserving existing data in the database.
```

Comment:
"real life" -> real world
"databases schema needs to be changed" -> "database schemas need to be updated"